stress: failed test in cockroach/cli/cli.test: TestNoLinkForbidden

Binary: cockroach/static-tests.tar.gz sha: https://github.com/cockroachdb/cockroach/commits/3f5b8dbe20282632850030143fa9ed8440cd2e53

Stress build found a failed test:

```
=== RUN   TestNoLinkForbidden
--- FAIL: TestNoLinkForbidden (0.00s)
	build.go:69: cannot find package "github.com/cockroachdb/cockroach" in any of:
			/usr/local/go/src/github.com/cockroachdb/cockroach (from $GOROOT)
			($GOPATH not set)
```

Run Details:

```
0 runs so far, 0 failures, over 5s
1 runs completed, 1 failures, over 6s
FAIL
```
Please assign, take a look and update the issue accordingly.

--- a/testutils/buildutil/build.go
+++ b/testutils/buildutil/build.go
@@ -58,12 +58,19 @@ func TransitiveImports(importpath string, cgo bool) (map[string]struct{}, error)
 // VerifyNoImports verifies that a package doesn't depend (directly or
 // indirectly) on forbidden packages. The forbidden packages are specified as
 // either exact matches or prefix matches.
+// If GOPATH isn't set, it is an indication that the source is not available and
+// the test is skipped.
 func VerifyNoImports(
 	t testing.TB,
 	pkgPath string,
 	cgo bool,
 	forbiddenPkgs, forbiddenPrefixes []string,
 ) {
+	// Skip test if source is not available.
+	if build.Default.GOPATH == "" {
+		t.Skip("GOPATH isn't set")
+	}
+
 	imports, err := TransitiveImports(pkgPath, true)
 	if err != nil {
 		t.Fatal(err)

